examples/kms/vault/csi-kms-connection-details.yaml

```diff
@@ -60,5 +60,12 @@ data:
       "KMS_SECRET_NAME": "ceph-csi-aws-credentials",
       "AWS_REGION": "us-west-2"
     }
+  kp-metadata-test: |-
+    {
+      "KMS_PROVIDER": "kp-metadata",
```

Comment:
IMO  `kp-metadata` is a not good identifier can it be `ibm-encryption`?

Reply:
ibm encryption has been done by many services, so `kp` stands better. thanks.